security/pkg/nodeagent/test/cert_rotation/rotation_test.go

```diff
@@ -72,8 +72,8 @@ func TestCertRotation(t *testing.T) {
 		numSSLConnError := stats[fmt.Sprintf("listener.127.0.0.1_%d.ssl.connection_error", setup.InboundListenerPort)]
 		numSSLVerifyNoCert := stats[fmt.Sprintf("listener.127.0.0.1_%d.ssl.fail_verify_no_cert", setup.InboundListenerPort)]
 		numSSLVerifyCAError := stats[fmt.Sprintf("listener.127.0.0.1_%d.ssl.fail_verify_error", setup.InboundListenerPort)]
-		numOutboundSDSUpdate := stats["cluster.outbound_cluster_tls.client_ssl_socket_factory.ssl_context_update_by_sds"]
-		numInboundSDSUpdate := len(certSet) - 1
+		numOutboundSDSRotate := stats["cluster.outbound_cluster_tls.client_ssl_socket_factory.ssl_context_update_by_sds"]
```

Comment:
numOutboundSDSRotate := stats["cluster.outbound_cluster_tls.client_ssl_socket_factory.ssl_context_update_by_sds"] - 1

Reply:
According to the context this stat of ssl_context_update_by_sds  just show the update count for the outbound. So think no needs to -1